deepreg/model/loss/util.py

```diff
@@ -23,3 +23,137 @@ def call(self, y_true: tf.Tensor, y_pred: tf.Tensor) -> tf.Tensor:
         :return: negated loss.
         """
         return -super().call(y_true=y_true, y_pred=y_pred)
+
+
+EPS = tf.keras.backend.epsilon()
+
+
+def rectangular_kernel1d(kernel_size: int) -> (tf.Tensor, tf.Tensor):
```

Comment:
as now we have so many kernels, do you think they worth a split `kernel.py` file, and we can also use REGISTRY ^^ so that no more if/else

Reply:
fine but another ticket